### ISSUE TITLE
Verbose feedback user warning if PP3 fails to save

### ISSUE THREAD

**Issue by verliba** [Beep6581/RawTherapee#4386]
Using version 5.3
Simple to reproduce.. write protect the folder with images.
Make settings to an image in the folder..
pp3 can't be saved of course..
But I don't know., I edit all the remaining 50 files, and quit RT at the end of the day..
What a surprise after restart RT does not remember any changes..

**Comment by heckflosse** [comment]
I agree that the behaviour could be impoved. Meanwhile as a workaround you could try this (just a guess, as I didn't test)

![grafik](https://user-images.githubusercontent.com/13733487/36067665-d67fac48-0ec1-11e8-91b4-67ae4d854347.png)

**Comment by Beep6581** [comment]
Related in the sense that we need a notification system:
https://github.com/Beep6581/RawTherapee/issues/4261
